tasks/integration.yml

```diff
@@ -46,6 +56,18 @@
   loop: "{{ datadog_integration|dict2items }}"
   when: item.value.action == "remove" and ansible_facts.os_family == "Windows"
 
+- name: Removing integrations (macOS)
```

Comment:
This is the same as on Linux, we don't need a macOS-specific job.

Reply:
you are right - removed this one and renamed the task above it to make it clear it runs on both unix-like and macOS